sdk/sdk-core/src/types/block.ts

```diff
@@ -0,0 +1,3 @@
+export type BlockNumber<unit = bigint> = unit;
+
+export type BlockTag = "latest" | "earliest" | "pending" | "safe" | "finalized";
```

Comment:
are all these tags applicable to Linea? if not, it might be worth having EthereumBlockTag and LineaBlockTag